src/test/org/nlogo/headless/TestCompiler.scala

```diff
@@ -147,8 +167,7 @@ class TestCompiler extends FunSuite with OneInstancePerTest with BeforeAndAfterE
   /// isReporter
 
   val reporters = Seq("3", "[]", "[", "((5))", "timer", "glob1")
-  val nonReporters = Seq("", ";", " ; ", "ca", "((ca))",
-                         "5984783478344387487348734", "gkhjfghkjfhjkg")
+  val nonReporters = Seq("", ";", " ; ", "ca", "((ca))", "5984783478344387487348734")
```

Comment:
This may be a particularly controversial consequence of removing the `forgiving` parameter, though its a natural result of allowing symbols in the language.